lib/expansion_rules.rb

```diff
@@ -65,6 +68,7 @@ def recurring
     { document_type: :need,                       fields: DEFAULT_FIELDS_WITH_DETAILS },
     { document_type: :finder, link_type: :finder, fields: DEFAULT_FIELDS_WITH_DETAILS },
     { document_type: :step_by_step_nav,           fields: DEFAULT_FIELDS_WITH_DETAILS },
+    { document_type: :role,                       fields: DEFAULT_FIELDS_WITH_DETAILS },
```

Comment:
Is there a way we can be more explicit in which fields we expand from the `details` hash here? Similar to how we define the fields for `:travel_advice`?